### ISSUE TITLE
Open for requests?

### ISSUE THREAD

**Issue by Gathaeryx** [manolitto/openvlex2#2]
Hi! I really love your solution for the risers, it seems the cleanest way to create spiraling stairs and the sort. But I was wondering if it would be possible for you to add 'half-steps' on top of the current slabs, so the stairs would look somewhat more realistic than the large stone OpenForge slabs.

**Comment by manolitto** [comment]
Yeah, sure. Happy about any feedback and suggestions for improvement.

So, @Gathaeryx, if I got you right you want parts like [this](https://github.com/manolitto/openvlex2/blob/main/stairs_and_risers/dungeon_stone/curved/dungeon_stone_stairs.curved.inch.riser.1%2B2x45%C2%B0x0.5.openvlex.radial.stl) with half the height?

**Comment by manolitto** [comment]
> Hi! I really love your solution for the risers, it seems the cleanest way to create spiraling stairs and the sort. But I was wondering if it would be possible for you to add 'half-steps' on top of the current slabs, so the stairs would look somewhat more realistic than the large stone OpenForge slabs.

@Gathaeryx  Take a look at this [quick POC](https://github.com/manolitto/openvlex2/blob/main/stairs_and_risers/dungeon_stone/curved/dungeon_stone_stairs.curved.inch.riser.1%2B2x45%C2%B0x0.25.openvlex.radial.stl). Is it what you are looking for?

**Comment by Gathaeryx** [comment]
Sorry for the late reply, but thanks for the quick response! I did actually intend a bit different step: What I meant was if you could design a step with half the depth and half the height, so it could be place on the one you linked first. I guess that results in the design from your second post, but with only half the angle?

Or maybe even those two joined together, resulting in a double step to be pinned on the base. My goal is to get more steps covering the same height, like 'real' winding stairs. I do like Devon's approach with the ~1 inch sized steps, so you can place a mini on them, but I've found that it looks a bit wonky when I've put a lot of them side by side.
